apps/integration/src/app/app.module.ts

```diff
@@ -7,7 +7,7 @@ import { AppComponent } from './app.component';
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, BrowserAnimationsModule, AppRoutingModule],
+  imports: [BrowserAnimationsModule, BrowserModule, AppRoutingModule],
```

Comment:
Since we're cherry-picking PRs between multiple versions now, it'd probably be good to avoid any changes that aren't directly related to the fix or feature of the current PR (so that we avoid merge conflicts between the branches).

Reply:
Good call. Fixing.